xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -5654,3 +5660,78 @@ void CVideoPlayer::SetUpdateStreamDetails()
 {
   m_messenger.Put(std::make_shared<CDVDMsg>(CDVDMsg::PLAYER_SET_UPDATE_STREAM_DETAILS));
 }
+
+
+void CVideoPlayer::NotifySubtitleUpdate(bool bStatus, bool bStreamInfo)
+{
+  CVariant data;
+  data["player"]["playerid"] = static_cast<int>(CServiceBroker::GetPlaylistPlayer().GetCurrentPlaylist());
+  if (bStatus)
+  {
+    data["property"]["subtitleenabled"] = m_processInfo->GetVideoSettings().m_SubtitleOn;
+  }
+  if (bStreamInfo)
+  {
+    int iStream = m_processInfo->GetVideoSettings().m_SubtitleStream;
+    SubtitleStreamInfo info;
+    GetSubtitleStreamInfo(iStream, info);
+    if (info.valid)
+    {
+      CVariant contentEntry(CVariant::VariantTypeObject);
+      contentEntry["index"] = iStream;
+      contentEntry["isdefault"] = (info.flags & StreamFlags::FLAG_DEFAULT) != 0;
+      contentEntry["isforced"] = (info.flags & StreamFlags::FLAG_FORCED) != 0;
+      contentEntry["isimpaired"] = (info.flags & StreamFlags::FLAG_VISUAL_IMPAIRED) != 0;
+      contentEntry["language"] = info.language;
+      contentEntry["name"] = info.name;
+      data["property"]["currentsubtitle"] = contentEntry;
+    } else if (!bStatus) return;
+  }
+  CServiceBroker::GetAnnouncementManager()->Announce(ANNOUNCEMENT::Player, "OnPropertyChanged", data);
+}
+
+void CVideoPlayer::NotifyAudioUpdate()
+{
+  AudioStreamInfo info;
+  int iStream = m_processInfo->GetVideoSettings().m_AudioStream;
```

Comment:
const int stream = m_processInfo... please. Const and the type is not used as part of the variable name.